testsuite/MDAnalysisTests/coordinates/test_netcdf.py

```diff
@@ -795,6 +795,30 @@ def test_write_AtomGroup(self, universe, outfile, outtop):
                                       self.prec,
                                       err_msg="unitcells are not identical")
 
+    @pytest.mark.parametrize('var, expected', (
+        ('coordinates', 'angstrom'),
+        ('time', 'picosecond'),
+        ('cell_lengths', 'angstrom'),
+        ('cell_angles', 'degree'),
+        ('velocities', 'angstrom/picosecond'),
+    ))
+    def test_writer_units(self, universe, outfile, var, expected):
+        """Tests that the writer adheres to AMBER convention units
+        TODO: switch to input trajectory that also has force information.
+        """
+        trr = mda.Universe(GRO, TRR)
```

Comment:
Can we change this to a smaller universe? I think one of the TRZ files has force information (see `test_trz`)

Reply:
Unfortunately the files in `test_trz` didn't seem to have any force information, the `DLP_CONFIG` file is quite small (single frame, 216 atoms).